src/cli/commands/owner.js

```diff
@@ -147,31 +175,12 @@ export const {run, setFlags, hasWrapper, examples} = buildSubCommands(
     },
 
     rm(config: Config, reporter: Reporter, flags: Object, args: Array<string>): Promise<boolean> {
-      return mutate(
-        args,
-        config,
-        reporter,
-        (username: string, name: string): Messages => ({
-          info: reporter.lang('ownerRemoving', username, name),
-          success: reporter.lang('ownerRemoved'),
-          error: reporter.lang('ownerRemoveError'),
-        }),
-        (user: Object, pkg: Object): boolean => {
-          let found = false;
-
-          pkg.maintainers = pkg.maintainers.filter((o): boolean => {
-            const match = o.name === user.name;
-            found = found || match;
-            return !match;
-          });
-
-          if (!found) {
-            reporter.error(reporter.lang('userNotAnOwner', user.name));
-          }
+      reporter.warn(`\`yarn owner rm\` is deprecated. Please use \`yarn owner remove\`.`);
+      return remove(config, reporter, flags, args);
+    },
 
-          return found;
-        },
-      );
+    remove(config: Config, reporter: Reporter, flags: Object, args: Array<string>): Promise<boolean> {
```

Comment:
I wanted to keep consistency with the `ls` changes.

I have noticed that I forgot to change
```
['add <user> [[<@scope>/]<pkg>]', 'rm <user> [[<@scope>/]<pkg>]', 'ls [<@scope>/]<pkg>']
```
Should I do it? Also, `ls` is still there, should I change it as part of another issue/PR?